packages/notification/channel/mailgun.ts

```diff
@@ -22,17 +22,24 @@
  * SOFTWARE.                                                                      *
  **********************************************************************************/
 
-import Mailgun from 'mailgun.js'
-import formData from 'form-data'
+import FormData from 'form-data'
 import Joi from 'joi'
 import type { NotificationMessage } from '.'
 import Mailgen from 'mailgen'
+import { sendHttpRequest } from '../utils/http'
 
-type NotificationData = {
+export type NotificationData = {
   apiKey: string
   domain: string
   recipients: string[]
-  username?: string
+}
+
+export type Content = {
```

Comment:
Could you elaborate why it needs to be exported?

Reply:
since `sendWithCustomContent(notificationData, content)` will be used by neosene, then the type of the parameter need to be imported there to be exactly the same